exec/rpc/src/main/java/org/apache/drill/exec/rpc/RemoteConnection.java

```diff
@@ -51,6 +51,22 @@
 
   SocketAddress getRemoteAddress();
 
+  void addSecurityHandlers();
+
+  boolean isEncrypted();
```

Comment:
Maybe `getEncryptionContext` to avoid adding setters and getters to this class if `EncryptionContext` is modified?